addons/account/models/account_journal.py

```diff
@@ -692,18 +692,19 @@ def create_document_from_attachment(self, attachment_ids=None):
             raise UserError(_("No attachment was provided"))
 
         invoices = self.env['account.move']
-        for attachment in attachments:
-            attachment.write({'res_model': 'mail.compose.message'})
-            decoders = self.env['account.move']._get_create_document_from_attachment_decoders()
-            invoice = False
-            for decoder in sorted(decoders, key=lambda d: d[0]):
-                invoice = decoder[1](attachment)
-                if invoice:
-                    break
-            if not invoice:
-                invoice = self.env['account.move'].create({})
-            invoice.with_context(no_new_invoice=True).message_post(attachment_ids=[attachment.id])
-            invoices += invoice
+        with invoices._disable_discount_precision():
```

Comment:
what's with invoices._disable_discount_precision() ? 
it certainly deserves a comment